### PR TITLE
Issue9 azraf

### PR DESCRIPTION
**Pull request by azraf-a**
Adding html restructuring to the profile page.

### CODE REVIEW

File: profilepage/templates/profilepage/user_profile.html [hunk 1]
```diff
@@ -56,24 +56,147 @@
...
+
+                    </div>
+                    <div class="column">
+
+                        <div class="box">
+                            <div class="field">
+                              <label for="" class="label">Days Created</label>
+                              <br>
+                              <div class="control">
+                                <input placeholder="Day 1" class="input is-primary" disabled>
```
Comment (sballan) [inline profilepage/templates/profilepage/user_profile.html:115]: Should these be input fields?  I don't think we'll want to accept any input here

File: profilepage/templates/profilepage/user_profile.html [hunk 2]
```diff
@@ -56,24 +56,147 @@
...
+                              <br>
+                              <div class="control">
+                                <input placeholder="Day 4" class="input is-primary" disabled>
+                              </div>
+                              <br>
+                        </div>
+
+                    </div>
+                </div>
+                <div class="columns is-centered">
```
Comment (sballan) [inline profilepage/templates/profilepage/user_profile.html:191]: I think this `div` might be extra

File: profilepage/templates/profilepage/user_profile.html [hunk 3]
```diff
@@ -56,24 +56,147 @@
...
+                  <div class="column is-5-tablet is-4-desktop is-3-widescreen">
+                  </div>
+                </div>
+              </div>
+
+    
+    <!-- <div class="container has-text-centered">
+    </div> -->
 
     <div class="hero-foot has-text-left">
```
Comment (sballan) [inline profilepage/templates/profilepage/user_profile.html:201]: I think this `div` might be extra

File: profilepage/templates/profilepage/user_profile.html (hunk 1 above)
Comment (azraf-a) [reply profilepage/templates/profilepage/user_profile.html:115]: Yep - lets them as input classes for clearer visibility during the presentation. When we build out day lists for users, I'll replace the control class div with one of Bulma's list setups. This is temporary.

**Pull request merged**